### PR TITLE
Export unsupported series error so it can be used by juju-core

### PR DESCRIPTION
**Pull request by wallyworld**
juju-core needs to create an unsupported series error, so export it.

### CODE REVIEW

File: charm.go [hunk 1]
```diff
@@ -88,6 +88,12 @@ func (e *unsupportedSeriesError) Error() string {
 	)
 }
 
+// NewUnsupportedSeriesError returns a error indicating that the requested series
```
Comment (axw) [inline charm.go]: an error

**Comment by axw** [comment]
LGTM with typo fixed

**Comment by wallyworld** [comment]
$$merge$$

**Comment by wallyworld** [comment]
:shipit:

**Pull request merged**